### PR TITLE
Avg ms/frame instead of FPS in performance overlay

### PR DESCRIPTION
**Pull request by liyuqian**
This change follows our trend to move all our performance metrics to frame time instead of FPS. Computing FPS from max ms/frame is misleading and we're no longer just using 60 FPS displays.

Tested on iOS devices. Android devices currently don't show any text because of https://github.com/flutter/flutter/issues/26387

We will add a gold test to the framework for performance overlay to both test this change and prevent regressions like https://github.com/flutter/flutter/issues/26387

Here's a screenshot of the new performance overlay:
image

### CODE REVIEW

**Comment by Hixie** [comment]
Can you post a screenshot of what it looks like?

File: flow/instrumentation.h [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 namespace flow {
 
+// DEPRECATED
+// The frame per second FPS could be different than 60 (e.g., 120).
 static const double kOneFrameMS = 1e3 / 60.0;
```
Comment (Hixie) [inline flow/instrumentation.h]: are there consumers that still exist that use this?

File: flow/layers/performance_overlay_layer.cc [hunk 2]
```diff
@@ -40,19 +40,14 @@ void VisualizeStopWatch(SkCanvas& canvas,
...
+    double max_ms_per_frame = stopwatch.MaxDelta().ToMillisecondsF();
+    double average_ms_per_frame = stopwatch.AverageDelta().ToMillisecondsF();
     std::stringstream stream;
     stream.setf(std::ios::fixed | std::ios::showpoint);
     stream << std::setprecision(1);
-    stream << label_prefix << "  " << fps << " fps  " << ms_per_frame
-           << "ms/frame";
+    stream << label_prefix << "  "
+           << "max " << max_ms_per_frame << " ms/frame, "
+           << "avg " << average_ms_per_frame << " ms/frame";
```
Comment (Hixie) [inline flow/layers/performance_overlay_layer.cc]: i'm a little conflicted about showing the average...

**Comment by Hixie** [comment]
test?

File: flow/instrumentation.h (hunk 1 above)
Comment (liyuqian) [reply flow/instrumentation.h]: It's used for drawing 3 horizontal lines: 16ms, 32ms, 48ms (https://github.com/flutter/engine/blob/master/flow/instrumentation.cc#L92)

Eventually, we should remove this altogether and get the refresh rate from https://github.com/flutter/engine/pull/7002 

I feel that it's better to do it in another PR to keep each one small and focused.

File: flow/layers/performance_overlay_layer.cc (hunk 2 above)
Comment (liyuqian) [reply flow/layers/performance_overlay_layer.cc]: There are often a few big spikes to make the max very different from the average (e.g., max is 300ms while the average is 11ms). One way of thinking is that average is more about the power consumption while max is more about janks.

**Comment by liyuqian** [comment]
See the edited PR description for a screenshot.

**Comment by liyuqian** [comment]
Ping...

**Pull request merged**